sdk/elastic/arm-elastic/src/models/index.ts

```diff
@@ -133,6 +128,8 @@ export interface MonitorProperties {
    * NOTE: This property will not be serialized. It can only be populated by the server.
    */
   readonly liftrResourcePreference?: number;
+  /** Flag to determine if User API Key has to be generated and shared. */
+  generateApiKey?: boolean;
```

Comment:
looks like the generateApiKey is changing from readonly to non-readonly Can you add this into features it's not a breaking I think